src/utils/isDev.ts

```diff
@@ -1,26 +1,30 @@
 const devHostNames = [/^local\.altinn\.cloud$/, /^\S+\.apps\.tt02\.altinn\.no$/];
 const studioHostNames = [/^dev\.altinn\.studio$/, /^altinn\.studio$/, /^studio\.localhost$/];
 
+let isDevCache: boolean | null = null;
+
 /**
  * Indicates whether the application is running in a development environment.
  * This can be either through LocalTest, altinn studio preview or TT02.
  */
-export function useIsDev(): boolean {
-  const isLocalOrStaging = useIsLocalOrStaging();
-  const isStudioPreview = useIsStudioPreview();
-  return isLocalOrStaging || isStudioPreview;
+export function isDev(): boolean {
```

Comment:
Nice, I have thought about doing this several times but never gotten around to it 😆